lib/repl.js

```diff
@@ -1283,8 +1292,15 @@ function complete(line, callback) {
             completionGroups.push(
               filteredOwnPropertyNames.call(this, contextProto));
           }
-          completionGroups.push(
-            filteredOwnPropertyNames.call(this, this.context));
+          let contextOwnNames =
+            filteredOwnPropertyNames.call(this, this.context);
+          if (vm.isContext(this.context)) {
+            // When the context is not `global`, builtins are not own
+            // properties of it.
+            ensureGlobalBuiltins();
+            contextOwnNames = globalBuiltins.concat(contextOwnNames);
```

Comment:
Just as a note since this is no hot code: it's almost always faster to use something like `array.push(...anotherArray)` than using `.concat()`. The reason is that no new array must be allocated.